cmd/flags.go

```diff
@@ -328,6 +351,21 @@ func ValidateNoArgs(ctx *cli.Context) error {
 	return nil
 }
 
+func ValidateNetworkFlags(ctx *cli.Context) error {
+
+	networkFlagsCount := 0
+	for _, flag := range NetworkFlags {
+		if ctx.IsSet(flag.Names()[0]) {
+			networkFlagsCount++
```

Comment:
Nit: You can already exit here if `networkFlagsCount > 1`. No need to iterate over all network flags then test the value out of the loop.

Reply:
makes sense